src/ElectricPokemon.java

```diff
@@ -0,0 +1,37 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class ElectricPokemon extends Pokemon {
+//   2 private final variables
+    private final String name;
+    private List<String> attacks = Arrays.asList("ThunderPunch", "ElectroBall", "Thunder", "VoltTackle");
+
+
+//    1 constructor
+public ElectricPokemon(String name, int level, int hp, String food, String sound) {
+    super(level, hp, food, sound);
+    this.name = name;
```

Comment:
"name" moet hier ook doorgegeven worden aan de super, anders krijgen we null waardes, wanneer dit de lijst geprint wordt in de PokemonGymImpl (die print functie gebruike namelijk "pokemon.getName()" en niet "electricpokemon.getName()")